### PR TITLE
PlexUpdate plugin updated for Plex Home

### PR DESCRIPTION
**Pull request by edcarroll**
Updated the PlexUpdate plugin to allow Plex Tokens to be provided with the
server details so that beets can update a Plex library that requires
authentication.

### CODE REVIEW

File: beetsplug/plexupdate.py [hunk 1]
```diff
@@ -44,6 +47,14 @@ def update_plex(host, port):
     return r
 
 
+def append_token(url, token):
+    """Appends the Plex Home token to the api call if required.
+    """
+    if token:
+        url += '?X-Plex-Token={0}'.format(token)
```
Comment (sampsyo) [inline beetsplug/plexupdate.py]: For safety, it's probably best to use:

    url += '?' + urllib.encode({'X-Plex-Token': token'})

**Comment by sampsyo** [comment]
Looks great, modulo the one small suggestion above! Would you mind adding to the documentation for the plugin (and a changelog entry)?

**Comment by edcarroll** [comment]
No problem - included your suggestion and updated the docs :)

**Comment by sampsyo** [comment]
Awesome. Thank you for adding this! :sparkles:

**Pull request merged**